addons/mail/static/src/models/suggested_partners/suggested_partners

```diff
@@ -0,0 +1,19 @@
+odoo.define('mail/static/src/models/suggested_partners/suggested_partners.js', function (require) {
```

Comment:
Just to avoid you some potential trouble, your file is currently missing the .js suffix.

Reply:
True, but it was already fixed on my local branch